.circleci/config.yml

```diff
@@ -79,8 +75,12 @@ commands:
             - workspace
   install_chrome:
     steps:
-      - browser-tools/install-chrome:
-          replace-existing: true
+      - browser-tools/install-browser-tools:
```

Comment:
For maintainability, can we make this list opt-in instead of opt-out by invoking `install-chrome` like before and then calling `install-chromedriver`? 

(My concern is that if `install-browser-tools` adds a new default browser in future, we'll implicitly install it on every VM.)

Reply:
browser-tools is version locked, so any additional browser will have to come from an explicit version upgrade. Is this still a concern in this case?

Reply:
Version locking addresses the implicit-install concern, but doesn't address the maintainability concern. E.g. When we add new steps to be run only on the VMs that test on Firefox, etc. (coming soon once IE 11 is deprecated), opt-in steps will read better than opt-out.

**opt-out:**
```yaml
  install_chrome:
    steps:
      - browser-tools/install-browser-tools:
          install-chrome: true
          install-chromedriver: true
          install-firefox: false
          install-geckodriver: false
          replace-existing-chrome: true
  install_firefox:
    steps:
      - browser-tools/install-browser-tools:
          install-chrome: false
          install-chromedriver: false
          install-firefox: true
          install-geckodriver: false
          replace-existing-chrome: false
```

**opt-in:**
```yaml
  install_chrome:
    steps:
      - browser-tools/install-chrome:
          replace-existing: true
      - browser-tools/install-chromedriver
  install_firefox:
    steps:
      - browser-tools/install-firefox
```

Reply:
I'm convinced. Done.